### DISCUSSION TITLE
NPC Roboto. Add options for mass converting

### DISCUSSION THREAD

**Discussion by Santaria94** [chromiecraft/chromiecraft#3445] [Suggestions]
Suggestion: Add options for mass converting for Roboto. For example 5, 10 and 20 items per click or something like that.

At the moment, to convert, you need to perform a huge number of actions.  For example, I want to make 5 enchants "Greater Fire Resistance" for auction. For each enchant, I need to convert items 12 times. Each converting takes 3 clicks (click on Roboto, select the desired converting and confirm the action). Total 5 * 12 * 3 = 180 clicks to convert items for 5 enchants. But that is not all. Items must be picked up from the mail. I use the "Postal" addon to open email messages, but because These are emails from Blizzard, it refuses to open them automatically. That's another 60 clicks to open the emails and 60 clicks to delete those emails, since they refuse to be automatically deleted too.

Total 180 + 60 + 60 = 300 clicks to get items for 5 "Greater Fire Resistance"...

**Comment by balleny** [comment]
Atleast the option to convert 1 item or 1 stack of the item (although this might be a bit tricky as some of the items have different stack sizes (10 or 20)

**Comment by 55Honey** [comment]
https://github.com/55Honey/Acore_ExchangeNpc/issues/4
